pkgpanda/__init__.py

```diff
@@ -225,7 +225,7 @@ def validate_compatible(packages, roles):
         ids.add(str(package.id))
         tuples.add((package.name, package.variant))
 
-    # All requires are met.
+    # All requirements are met.
```

Comment:
This is actually meant to be requires, as thats the key which is used in the json